server/src/main/scala/org/scassandra/server/priming/prepared/PrimePreparedPatternStore.scala

```diff
@@ -21,12 +21,12 @@ import com.typesafe.scalalogging.LazyLogging
 import org.scassandra.server.cqlmessages.Consistency
 import org.scassandra.server.priming.query.{Prime, PrimeCriteria, PrimeMatch}
 import org.scassandra.server.priming.routes.PrimingJsonHelper
-import org.scassandra.server.priming.{Defaulter, PrimeAddResult, PrimeAddSuccess}
+import org.scassandra.server.priming.{PrimeAddResult, Defaulter, PrimeAddSuccess}
 import org.scassandra.server.priming.json.Success
 
 import scala.concurrent.duration.FiniteDuration
 
-class PrimePreparedPatternStore extends LazyLogging with PreparedStore with PreparedStoreLookup {
+class PrimePreparedPatternStore extends LazyLogging with PreparedStore[PrimePreparedSingle, PreparedPrime] with PreparedStoreLookup {
```

Comment:
Switch LazyLogging and PreparedStore, doesn't make any difference but is a more accurate description of the type